src/hexo/themes/documentation/source/core/css/about.css

```diff
@@ -0,0 +1,16 @@
+/*
+ * ABOUT + SUB-PAGES
+ * =============================================
+ * @Dependencies: base.css, table-lists.css, type.css
+ */
+
+/* Contributors Page */
+
+th sub {
+    font-size: 1.6rem;
+}
+
+.contributors th img {
+    border-radius: 50px;
```

Comment:
This style only applies to the first row. The second row is `tbody` instead of `th`.![contributor](https://user-images.githubusercontent.com/20218531/27496185-dbdb6200-5809-11e7-99e6-90524a53d2e8.PNG)